### PR TITLE
adding dns name label support for azure container instance .net sdk

### PR DESCRIPTION
**Pull request by xizha162**
## Description
With this change, user can now specify a dns name label for the container they create in ACI using Azure .Net SDK.
The only semantics changes are in ipaddress.cs (to include the new property) and tests. Changes in other files are mostly format changes from AutoRest

Link to the PR for rest api spec:
https://github.com/Azure/azure-rest-api-specs/pull/2309
 A simple bug fix (casing) to the spec:
https://github.com/Azure/azure-rest-api-specs/pull/2410

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by xizhamsft** [comment]
@shahabhijeet ,  could you take a look of this change? It is very simple. I am just adding new property (dns name label) for the container instance.

File: src/SDKs/ContainerInstance/Management.ContainerInstance/Microsoft.Azure.Management.ContainerInstance.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 	<PropertyGroup>
 		<Description>Microsoft Azure Container Instance Management Library</Description>
-		<Version>1.0.1-preview</Version>
+		<Version>1.0.2-preview</Version>
```
Comment (shahabhijeet) [inline src/SDKs/ContainerInstance/Management.ContainerInstance/Microsoft.Azure.Management.ContainerInstance.csproj]: Missing < PackageReleaseNotes >. Add release notes as to what is being fixed/added to this release.

**Review by shahabhijeet — COMMENTED** [review]
Missing link to your swagger spec PR
Missing meta data file that get's updated when you use generate.cmd to generate

File: src/SDKs/ContainerInstance/Management.ContainerInstance/Microsoft.Azure.Management.ContainerInstance.csproj (hunk 1 above)
Comment (xizhamsft) [reply src/SDKs/ContainerInstance/Management.ContainerInstance/Microsoft.Azure.Management.ContainerInstance.csproj]: @shahabhijeet , fixed. Please take a look.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**